docs/content/release_notes/changelog_1.8.0.md

```diff
@@ -5,6 +5,5 @@ IBM® block storage CSI driver 1.8.0 added new support and enhancements:
     - New HyperSwap support
     - New NVMe®/FC support
     - Increased StorageClass `SpaceEfficiency` parameter capabilities
-- Added custom resource configurability for the CSI driver health port
```

Comment:
why was this removed?

Reply:
because it's in the 1.8.0 file and not 1.9

Reply:
I'm pretty sure the health port configurability was indeed [part of 1.8.0](https://github.com/IBM/ibm-block-csi-operator/blob/v1.8.0/config/samples/csi.ibm.com_v1_ibmblockcsi_cr.yaml#L83), have anyone said otherwise?

Reply:
fixed